examples/sw/simple_system/common/simple_system_common.h

```diff
@@ -95,4 +95,19 @@ void timer_disable(void);
  */
 uint64_t get_elapsed_time(void);
 
+/**
+ * Enables/disables the instruction cache. This has no effect on Ibex
+ * configurations that do not have an instruction cache and in particular is
+ * safe to execute on those configurations.
+ *
+ * @param enable if non-zero enables, otherwise disables
+ */
+static inline void icache_enable(int enable) {
+  if (enable) {
+    asm volatile("csrs 0x7c0, 1");
+  } else {
+    asm volatile("csrc 0x7c0, 1");
```

Comment:
```suggestion
    asm volatile("csrc 0x7c0, 0");
```

Reply:
It should be 1 in both cases, the first instruction is csr**s** (set a CSR bit) and the second is csr**c** (clear a CSR bit).

Reply:
Very easy to miss this one letter difference. Would it make sense to put this in a comment?

Reply:
Done